src/enum/ColorTheme.ts

```diff
@@ -0,0 +1,29 @@
+export enum ColorTheme {
+	PRIMARY = 'primary',
```

Comment:
OK, I'll change it to PascalCase.

> And do we need because of some specific reason to use the custom values of the enums?

Well, yeah, I think actually we do. Because for example I use enum for component sizes. In the components there's a prop `size` for which you should use ComponentSize.Large, ComponentSize.Medium etc. but if I don't use custom values, the values will be 0, 1... and I want the developers to be able to write not only `size={ComponentSize.Large}` but also if they want to simply `size="large"`.

Also, in the theme file where I use enums like this

```
    borderRadius: {
		[ComponentSize.SMALL]: '4px',
		[ComponentSize.MEDIUM]: '5px',
		[ComponentSize.LARGE]: '6px'
    },
    spacing: {
		[Spacing.XXS]: 4,
		[Spacing.XS]: 8,
    (...)
```

it's better to have enums with values because then when you print it out you will have

```
borderRadius: {
    small: '4px',
    medium: '5px',
    large: '6px'
}
```

instead of

```
borderRadius: {
    0: '4px',
    1: '5px',
    2: '6px'
}
```

Reply:
Sorry, I take it back. I realized it doesn't work like that. You cannot write simply "large" for ComponentSize.Large even if the value is the same. But it would be nice, I mean, I don't want to force developers to use those enums if they don't want to. That seems silly to me. I thought it would be only optional.

Reply:
Sure, good catch! I think string literals will be the best. 👏  Let's do it.